docs/data-collection/preliminary.md

```diff
@@ -129,13 +128,23 @@ This block describes how to prepare a laptop with a running *Psychopy 3* install
     ```
 - [ ] Install bdist_mpkg, py2app and attrdict:
     ```
-    pip3 attrdict py2app bdist_mpkg
+    pip3 install attrdict py2app bdist_mpkg
+    ```
+- [ ] Install wxPython. Careful to replace the ubuntu version with the one corresponding to your system (you can check which Ubuntu version is installed on your system by running `lsb_release -a` in the terminal)
+    ```
+    pip3 install -U \
+        -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-20.04 \
```

Comment:
I'm sure we will forget replacing the version. Let's not allow that degree of freedom.

```suggestion
        -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-$( lsb_release -r -s ) \
```